### ISSUE TITLE
PacBio Tutorial

### ISSUE THREAD

**Issue by LPeit** [benjjneb/dada2#566]
Hi! I was wondering if there will be a Tutorial on how to use Dada2 1.9 on PacBio Full-Length 16S Amplicon data? That would be really great and helpful! :) Thanks!

**Comment by benjjneb** [comment]
Yes there will be, but for now you can follow the reproducible analyses scripts from the preprint which cover all the basics. Scroll down and follow the links in the github repo Readme: https://github.com/benjjneb/LRASManuscript

**Comment by LPeit** [comment]
That's Great! Thanks for the quick answer!
